summoner.cabal

```diff
@@ -65,7 +66,7 @@ library
                      , process
                      , text
                      , time
-                     , tomland ^>= 0.3.1
+                     , tomland ^>= 0.4.0
                      , relude ^>= 0.1.1
```

Comment:
I think we can also migrate to `relude ^>= 0.2.0`. This should be an easy migration.